desktop/analytics/bidding.js

```diff
@@ -81,7 +81,17 @@ $(document).on('click', '.my-active-bids-bid-button', function (e) {
 })
 
 // Clicked "Bid" (context_type: artwork page)
-$(document).on('click', '.artwork-auction__bid-form__button', function () {
+$(document).on('click', 'button.artwork-auction__bid-form__button', function () {
```

Comment:
These two tracking calls look very similar-- is there a reason we're rendering an `a` vs a `button` in some cases? Would it be possible to streamline?